### PR TITLE
Ian

### PR DESCRIPTION
**Pull request by ianick87**
Checked the serverless function and bucket examples and made some changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,6 +7,11 @@ about the project. The project is currently being developed using Hugo based on
 
 The project follows the general structure of a `hugo` project, and to work with the codebase, you will need to [install hugo v0.93.3](https://gohugo.io/getting-started/installing/) (note if downloading the binary directly, you need to download the extended version to get live SCSS reload). Once installed, you should be able to run `hugo server` to run a local version of the project. 
 
+## Install steps
+wget https://github.com/gohugoio/hugo/releases/download/v0.93.3/hugo_0.93.3_Linux-64bit.deb
+sudo dpkg -i hugo_0.93.3_Linux-64bit.deb
+then make your directory, clone cdev-website then cd into it
+hugo server
```
Comment (Pia007) [inline README.md:14]: Consider using three backticks, ```, for each of lines 11 thru 14.

File: content/docs/examples/functions.md [hunk 2]
```diff
@@ -76,7 +76,7 @@ cdev run function -h
 
 {{<break 1>}}
 ## Responding to Events
-One of the most important aspects of Serverless Functions is that they can triggered by different `Events`. By passing an `Event` construct into the args of your `Serverless Function`, your deployed `Serverless Function` will be triggered by the event in the Cloud. The following demonstrates a `Serverless Function` being hooked up to an Api `Route Event`.
```
Comment (Pia007) [inline content/docs/examples/functions.md:79]: change Api to `API`

File: content/docs/examples/functions.md [hunk 3]
```diff
@@ -76,7 +76,7 @@ cdev run function -h
...
+One of the most important aspects of Serverless Functions is that they can be triggered by different `Events`. By passing an `Event` construct into the args of your `Serverless Function`, your deployed `Serverless Function` will be triggered by the event in the Cloud. The following demonstrates a `Serverless Function` being hooked up to an Api `Route Event`.
```
Comment (Pia007) [inline content/docs/examples/functions.md:79]: change Api to `API`

File: content/docs/examples/functions.md [hunk 4]
```diff
@@ -76,7 +76,7 @@ cdev run function -h
...
 {{<codesnippet `/source_code/function_examples/event_function.py`>}}
 
 You can now get the url of the created `Api` and execute your function by going to the `/hello` route.
```
Comment (Pia007) [inline content/docs/examples/functions.md:82]: change Api to `API`

**Review by Pia007 — CHANGES_REQUESTED** [review]
Hi Ian, I have reviewed your PR and I have suggested some minor changes. I think that @DanielSanchez97 @jtfrench23 will have more insight to the code snippets.

**Comment by ianick87** [comment]
Hey Pia, I made the changes you suggested and I'm going to add Daniel to review the PR. Thank you!

File: README.md [hunk 5]
```diff
@@ -8,10 +8,10 @@ about the project. The project is currently being developed using Hugo based on
 The project follows the general structure of a `hugo` project, and to work with the codebase, you will need to [install hugo v0.93.3](https://gohugo.io/getting-started/installing/) (note if downloading the binary directly, you need to download the extended version to get live SCSS reload). Once installed, you should be able to run `hugo server` to run a local version of the project. 
 
 ## Install steps
-wget https://github.com/gohugoio/hugo/releases/download/v0.93.3/hugo_0.93.3_Linux-64bit.deb
-sudo dpkg -i hugo_0.93.3_Linux-64bit.deb
-then make your directory, clone cdev-website then cd into it
-hugo server
+```wget https://github.com/gohugoio/hugo/releases/download/v0.93.3/hugo_0.93.3_Linux-64bit.deb
```
Comment (Pia007) [inline README.md:11]: Hi Ian, I am sorry for any confusion. I meant for Line 11, 12 & 14, wrap each line in 3 backticks to denote a code block. I have included an image of an example below.   
11
[image](https://raw.githubusercontent.com/cdev-framework/assemblit/main/CONTRIBUTING.md?token=GHSAT0AAAAAABH4LER5TNJWQLGUO3FLYRFOYWO2VWA)

**Review by Pia007 — CHANGES_REQUESTED** [review]
Hi Ian, sorry for any confusion. My comment contains a better explanation.

**Review by Pia007 — APPROVED** [review]
Looks good, Ian!

**Review by DanielSanchez97 — APPROVED** [review]
Looks good to me (LGTM). Only thing is next time make the branch name descriptive of what the changes are.

**Pull request merged**